pages/index.tsx

```diff
@@ -18,7 +18,7 @@ const CallToActionButton = () => {
         <div>
             <Link href={redirectUrl}>
                 <button className="w-full py-8 px-8 md:py-4 md:px-12 rounded-3xl bg-blue-600 hover:bg-blue-700 focus:ring-blue-500 focus:ring-offset-blue-200 text-white transition ease-in duration-200 text-center text-base font-semibold shadow-md focus:outline-none focus:ring-2 focus:ring-offset-2">
-                    I nak SMS when appointment is ready{' 💬 '}
+                    Jom! Subscribe to NakVaksin!
```

Comment:
I wonder if first time user seeing this, they must think about "Subscribing" like Subscribing to Netflix or youtube and hesitate to click because they might think need to pay? 

WDYT?

Reply:
if you think ok then I don't mind haha xDD